app/services/authentication.rb

```diff
@@ -0,0 +1,45 @@
+class Authentication
```

Comment:
What about calling it `AuthenticationService`? `Authentication` feels too generic.

Reply:
We don't do that with any of our other service objects, so I opted for consistency.

Reply:
I think this is a mistake, personally. I think there's way too much opportunity to clash. In Rails it's only the models that don't have a suffix ...

Reply:
Yeah. I can rename this one and put another PR through to rename the others.